requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 # test dependencies
 pytest>=2.8.2
+pytest-depends>=1.0.1
```

Comment:
I added this pytest module because case management test cases depends on each other. Here is some background.

Creating new cases is an expensive task. It will be especially bad if these integration tests create a bunch of test cases in production. Our support org asked us to limit the number of test cases because it skews various reports based on the case data.

Most test cases require a case to take upon an action. Instead of creating a new case for each case, I opted for creating one case and reuse it.